Libplanet/Address.cs

```diff
@@ -186,6 +187,22 @@ public override string ToString()
             return $"0x{ToHex()}";
         }
 
+        private static string ToChecksumAddress(string hex)
+        {
+            byte[] bytes = Encoding.ASCII.GetBytes(hex);
```

Comment:
What if the given `hex` has any characters out of ASCII character set?  This case should be addressed.  See also `ByteUtil.ParseHex()` method's code.

Reply:
Thanks, I'll add code for the case.

Reply:
When given `hex` has some non-ASCII characters, `Encoding.ASCII.GetBytes` doesn't raise an exception, but `ByteUtil.ParseHex` raise `FormatException` instead. So I caught it and re-raised `ArgumentException`.